disco_aws_automation/disco_elastigroup.py

```diff
@@ -434,32 +434,33 @@ def delete_all_recurring_group_actions(self, hostclass=None, group_name=None):
     def create_recurring_group_action(self, recurrance, min_size=None, desired_capacity=None, max_size=None,
                                       hostclass=None, group_name=None):
         """Creates a recurring scheduled action for a hostclass"""
-        existing_group = self.get_existing_group(hostclass=hostclass, group_name=group_name)
-
-        task = {
-            'cronExpression': recurrance,
-            'taskType': 'scale'
-        }
+        existing_groups = self.get_existing_groups(hostclass=hostclass, group_name=group_name)
+        for existing_group in existing_groups:
```

Comment:
trying to be consistent with https://github.com/amplifylitco/asiaq/blob/master/disco_aws_automation/disco_autoscale.py#L568

I don't think this would ever happen at the moment though. But if it did happen during blue/green then I don't see why we wouldn't want both groups to have the same schedule